src/components/adaptive-cards/Action/ActionOpenURL/ActionOpenUrl.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {isValidUrl} from '../../../../util';
+import {acPropTypes, registerComponent} from '../../Component/Component';
+import webexComponentClasses from '../../../helpers';
+import Action from '../Action';
+
+/**
+ * Adaptive Cards Action.OpenUrl component
+ * https://adaptivecards.io/explorer/Action.OpenUrl.html
+ *
+ * @param {object} props  React props passed to the component
+ * @param {object} props.data  Active cards definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Custom style to apply
+ * @returns {object} JSX of the component
+ */
+export default function ActionOpenUrl({data, className, style}) {
+  const [cssClasses] = webexComponentClasses('ac-action-open-url', className);
+  const isValidHttpUrl = isValidUrl(data.url, 'https:', 'http:');
```

Comment:
protocols should be an array with 2 items

Reply:
yes, I missed the square brackets